### ISSUE TITLE
MidiNet training labels

### ISSUE THREAD

**Issue by nivedr** [RichardYang40148/MidiNet#2]
I am having some trouble understanding what are the labels that have been used while training the network. Specifically I am interested to know what you have used alongside the training data directories in **1e6bf06**: MidiNet/v1/model.py, line 136
Would appreciate it if you could add any information on the _structure_ of the dataset, so that others can build their own without touching your implementation

**Comment by RichardYang40148** [comment]
Please check the paper, thanks
